snapshot/pkg/controller/snapshotter/snapshotter.go

```diff
@@ -372,73 +372,60 @@ func (vs *volumeSnapshotter) getPlugin(snapshotName string, snapshot *crdv1.Volu
 }
 
 func (vs *volumeSnapshotter) getSnapshotStatus(snapshot *crdv1.VolumeSnapshot) (string, *crdv1.VolumeSnapshotData, error) {
```

Comment:
looks like we can combine getSnapshotStatus and getSimplifiedSnapshotStatus after this change. both functions are doing the similar things.

Reply:
Yes.